apache2/waf_lock/waf_lock.cpp

```diff
@@ -0,0 +1,512 @@
+#include"waf_lock_external.h"
+#include"waf_lock_internal.h"
+
+
+/**
```

Comment:
Minor nit, can we just use `//` instead of `/*` everywhere for comments? Just for consistentcy?

Reply:
think about it, for block of comments, /* might look better